telemetry/definitions/commonDefinitions.json

```diff
@@ -159,6 +159,17 @@
             "type": "string",
             "allowedValues": ["IAM", "SecretsManager"],
             "description": "How the database credentials are being retrieved"
+        },
+        {
+            "name": "snippetPrefix",
```

Comment:
Why do we need this? why not just name?

Reply:
Yeah I considered name. No problem either way. Name makes more sense if it's trying to be generic with something like Live Templates. Prefix makes more sense if we use the vscode terminology.